### ISSUE TITLE
VISSL & Detectron Installation Instruction Conflict

### ISSUE THREAD

**Issue by ysharma1126** [facebookresearch/vissl#517]
Minor note, the torch version recommended to install in the [VISSL instructions](https://github.com/facebookresearch/vissl/blob/main/INSTALL.md#step-2-install-pytorch-pip) appears to be incompatible with the object detection benchmark dependency Detectron2, which appears to require PyTorch ≥ 1.8, see [here](https://github.com/facebookresearch/detectron2/blob/main/INSTALL.md#requirements).

**Comment by QuentinDuval** [comment]
Hi @ysharma1126,

Indeed, good catch!

I think we could update the PyTorch version to 1.8 and higher (we do work with VISSL on our side with higher versions like 1.8, 1.9 or more). I will delegate this question to @iseessel though, since he is more knowledgable on the packaging / dependencies than me.

_Side note: Once thing I am not 100% sure is whether it is a good idea anyway to use the same conda environment for detectron2 and VISSL (since it introduces some coupling of dependency versions). Although we do have a script to run detectron2, I must admit I never played with object detection for now, maybe we could find a better way._

Back on the point @ysharma1126, since @iseessel is off this week, the response might take some time.

Thank you again for raising this :)
Quentin

**Comment by ysharma1126** [comment]
No worries, no urgency on my end :)

**Comment by iseessel** [comment]
@ysharma1126 Yes you should be able to update the pytorch version to whatever version is compatible with Detectron. Let me know if you need any guidance on how to do this.
